File: 단계별로_풀어보기/1. 입출력과 사칙연산/2_1000.js
/**
 * 1-2. 1000. A+B
 *
 * 두 수를 더한 값을 출력하는 문제이다.
 *
 * Node 로 코딩 테스트를 하는 상황을 대비하기 위해 연습겸 풀어봤다.
 */

const fs = require('fs')
const input = fs.readFileSync('/dev/stdin').toString().split(' ')
const a = parseInt(input[0])
const b = parseInt(input[1])

process.stdout.write((a + b).toString())

File: 1. 단계별로_풀어보기/1~9/1. 입출력과 사칙연산/2_1000.py
# 1-2. 1000. A+B
#
# 두 수를 더한 값을 출력하는 문제이다.
# 빠른 출력을 위해 output 에 f 로 포매팅을 해줬다.

import sys

input = sys.stdin.readline
output = sys.stdout.write

A, B = map(int, input().split())

output(f"{A + B}")